src/main/java/Reporter.java

```diff
@@ -11,7 +11,39 @@
  */
 public class Reporter {
     public static final String REPORT_SUFFIX = "_Report.txt";
+    public static final int REPORT_ALL          = 0xFFFFFFFF;
+    public static final int NUM_PROJECTS        = 0b1;
```

Comment:
They will end up getting longer.  Part of what I like about this approach is that the old flags can be totally unaffected by adding new flags (up to 32 of them).